### DISCUSSION TITLE
Random Mode

### DISCUSSION THREAD

**Discussion by erikmackinnon** [some9000/StylePile#12] [Ideas]
Hi! Love the new release + updates, looking good.

What are your thoughts on having a "random" setting in each dropdown so that the script just chooses a mood, colors, view, etc. at random from the picklist?

It would be great to cycle a batch through various options to see what the outputs are.

**Comment by some9000** [comment]
Glad to hear that you enjoy it.
That is actually a pretty cool idea. One current downside is that I have no idea how to get the prompt to show under the preview, like the regular version does. Then again, it only shows the main prompt anyway. Current plan is to implement 3 selections for both artists and styles with strength sliders. Having any of that be random might produce some wild results indeed.
Actually, this gives an idea to add **Random** and **All** options. Not everyone will want to see results by 135 artists, but who knows. Although it would be a pain to implement. Will certainly look into it.

**Reply by some9000** [reply]
Ok, current beta has a version of this and more implemented. You can mix and match random and non random artists, have random styles, make sets with different random/sequential elements. Basically a powerful generation playground. Feel free to check it out.

My thought about having something like automatically going through lists was a little extreme, but will probably add full lists to the help tabs. So users can copy paste all, nearly 300 artists, for example, into a batch field.

**Reply by manolofloyd** [reply]
This is great work indeed, just hope it could have a preset mode for all-random and not having to choose all categories every time. Thanks
